account_analytic_profit_loss_analysis/models/account_analytic_line.py

```diff
@@ -6,94 +6,86 @@
 
 
 class AccountAnalyticLine(models.Model):
-    _inherit = 'account.analytic.line'
+    _inherit = "account.analytic.line"
 
     # we cannot make this field required at model level because that would
     # interfere the record creation from HrTimesheetSheet thru JS
     analytic_type_id = fields.Many2one(
-        'analytic.type',
-        string='Analytic Type',
+        "analytic.type",
+        string="Analytic Type",
         # required=True,
     )
     related_analytic_type = fields.Selection(
-        related="analytic_type_id.type",
-        readonly=True,
+        related="analytic_type_id.type", readonly=True,
     )
     pj_id = fields.Many2one(
-        'project.project',
-        string='Related Project',
-        compute='_compute_pj_id',
-        inverse='_set_account_id',
+        "project.project",
+        string="Related Project",
+        compute="_compute_pj_id",
+        _inverse_="_set_account_id",
```

Comment:
@Ryoko04 Can you please check if this method is triggered as expected? Haven't seen `_inverse_` thing before.

@TimLai125 Is this a valid change?

Reply:
@yostashiro 
Yes, I should have changed  to 'inverse="_inverse_set_account_id"'. I noticed it now.
![image](https://user-images.githubusercontent.com/70553601/98630786-3dc33280-235f-11eb-92d3-35a42c934cd0.png)